locations/spiders/ubb_bg.py

```diff
@@ -0,0 +1,116 @@
+import re
+
+import scrapy
+from scrapy import Spider
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.hours import DAYS_BG, OpeningHours, day_range, sanitise_day
+from locations.items import Feature
+from locations.user_agents import FIREFOX_LATEST
+
+
+class UbbBGSpider(Spider):
+    name = "ubb_bg"
+    item_attributes = {"brand": "Обединена българска банка", "brand_wikidata": "Q7887555"}
+    start_urls = ["https://www.ubb.bg/offices/pins"]
+    user_agent = FIREFOX_LATEST
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+        "DEFAULT_REQUEST_HEADERS": {"Accept": "application/json, text/javascript, */*"},
+    }
```

Comment:
Do we need to ignore robots.txt? It doesn't look like we're blocked.

https://www.ubb.bg/robots.txt

Also headers are probably not needed.

```suggestion
```